eslint.config.js

```diff
@@ -1,6 +1,8 @@
 'use strict'
 
 module.exports = require('./')({
+  globals: ['assertsFoo'],
+  env: ['mocha'],
```

Comment:
`globals.mocha` can't be used here? it's ESM an only dep?

Reply:
This is simply to verify that it will be passed correctly, I should add a comment to highlight that, mocha is not used in this project

Reply:
Oh, I see what you mean now @jerome-benoit – `neostandard` does the `env` resolving built in